packages/editor-ui/src/views/MfaSetupView.vue

```diff
@@ -0,0 +1,168 @@
+<template>
```

Comment:
There is a lot of code duplication between this and `MfaSetupModal`. Maybe we should extract that code into a component.

Reply:
Yes, this view was an early iteration and ended up moving everything to a modal, which is why they are similar. Removed it already.